client/src/components/user_reviews/user_reviews.jsx

```diff
@@ -3,14 +3,66 @@ import UserReviewsEntry from '../user_reviews_entry/user_reviews_entry.jsx';
 const React = require('react');
 
 
-const UserReviews = (props) => {
-  return (
-    <ul>
-      {props.data.map((data) => {
-        return <UserReviewsEntry key={props.data.indexOf(data)} review={data} keywords={props.keywords} />;
-      })}
-    </ul>
-  );
-};
+class UserReviews extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      page: 1,
+    };
+    this.handleClick = this.handleClick.bind(this);
```

Comment:
I understand what you're doing here and with pageButton but I think it would be better to bind this at each passing rather than as a property.